src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VuapVideos.java

```diff
@@ -0,0 +1,32 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.ad;
+
+public class VuapVideos {
+
+  private VuapVideos() {
+    throw new IllegalAccessError("Utility class");
+  }
+
+  public static final String PORVATA_VAST = "porvata_vast";
+
+  public static final String VAST_VIDEO = "', '<?xml version=\"1.0\" encoding=\"UTF-8\"?><VAST xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance" +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)